draft-ietf-quic-transport.md

```diff
@@ -4186,12 +4186,13 @@ initial_max_streams_uni (0x0009):
   parameter is equivalent to sending a MAX_STREAMS ({{frame-max-streams}}) of
   the corresponding type with the same value.
 
-ack_delay_exponent (0x000a):
+ack_delay_multiplier (0x000a):
 
-: The ACK delay exponent is an integer value indicating an
-  exponent used to decode the ACK Delay field in the ACK frame ({{frame-ack}}).
-  If this value is absent, a default value of 3 is assumed (indicating a
-  multiplier of 8). Values above 20 are invalid.
+: The ACK delay multiplier is an integer value indicating a multiplier used to
+  decode the ACK Delay field in the ACK frame ({{frame-ack}}). If this value is
+  absent, a default value of 10 (corresponding to a resolution of 10
+  microseconds)is assumed. Values above 1,000,000 (corresponding to a resolution
+  of 1 second) are invalid.
```

Comment:
It might be better to leave the multiplier at 1 rather than pick a value.

Also, I assume that we want to prohibit 0.